src/ansys/fluent/core/quantity.py

```diff
@@ -0,0 +1,229 @@
+import pint
+from pint import Unit
+
+unit = pint.UnitRegistry(autoconvert_offset_to_baseunit=True, system="SI")
+quantity = unit.Quantity
+
+restricted_conversions = (
+    ("Hz", "rad/s"),
+    ("Hz", "radian/s"),
+    ("Hz", "rpm"),
+    ("Hz", "rps"),
+    ("Hz", "cps"),
+    ("hertz", "rad/s"),
+    ("hertz", "radian/s"),
+    ("hertz", "rpm"),
+    ("hertz", "rps"),
+    ("hertz", "cps"),
+    ("rpm", "Hz"),
+    ("rpm", "hertz"),
+    ("rpm", "rad/s"),
+    ("rpm", "radian/s"),
+    ("rpm", "rps"),
+    ("rpm", "cps"),
+    ("rps", "Hz"),
+    ("rps", "hertz"),
+    ("rps", "rad/s"),
+    ("rps", "radian/s"),
+    ("rps", "rpm"),
+    ("rps", "cps"),
+    ("cps", "Hz"),
+    ("cps", "hertz"),
+    ("cps", "rad/s"),
+    ("cps", "radian/s"),
+    ("cps", "rpm"),
+    ("cps", "rps"),
+    ("rad/s", "Hz"),
+    ("rad/s", "hertz"),
+    ("rad/s", "rpm"),
+    ("rad/s", "rps"),
+    ("rad/s", "cps"),
+    ("radian/s", "Hz"),
+    ("radian/s", "hertz"),
+    ("radian/s", "rpm"),
+    ("radian/s", "rps"),
+    ("radian/s", "cps"),
+)
+
+restricted_unit_expansion = {
```

Comment:
Later version would build this list automatically using a standard list of prefixes.

Reply:
Yes, sure. Now we have confirmed the usage of standard prefixes while writing test cases so we can generate list using them.

Reply:
Done.